app/views/catalog/access_panels/_location.html.erb

```diff
@@ -23,7 +23,15 @@
               <%= content_tag(:div, class: 'bound-with-note note-highlight') { render document.bound_with_note } %>
             <% else %>
               <% if location_level_request_link?(library, location) %>
-                <%= link_to_request_link(document: document, library: library, callnumber: location.items.first, class: 'btn btn-default btn-xs request-button pull-right') %>
+                <% if library.hoover_archive? %>
```

Comment:
`:sad_kitty:.jpg` but is there a pattern existing already in SW that could help w/ this stuff?